config/webpack.default.config.js

```diff
@@ -117,6 +118,17 @@ function getConfig(opts) {
     opts.env === "production"
       ? getProductionConfig(opts)
       : getDevelopmentConfig(opts);
+  const includeLoadablePlugin = () => {
+    if (opts.loadableConfig && Object.keys(opts.loadableConfig).length > 0) {
+      const LoadablePlugin = require("@loadable/webpack-plugin");
```

Comment:
change casing

Reply:
since it is a class we can't change the casing